dockers/docker-lldp-sv2/lldpmgrd

```diff
@@ -160,8 +160,9 @@ class LldpManager(object):
         while True:
             (state, c, fd) = sel.select()
             (key, op, fvp) = sst.pop()
+            fvp_dict = dict(fvp)
 
-            if len(fvp) == 1 and fvp[0][0] == "state" and fvp[0][1] == "ok":
+            if op == "SET" and fvp_dict["state"] == "ok":
```

Comment:
Are you sure you always have "state" in fvp_dict?
Probably better 
```
"state" in fvp_dict and fvp_dict["state"] == "ok"
```
?

Reply:
It should always be there, but it can't hurt to check. will update shortly.